Birthday-2024-Project/Scripts/UI/Settings/AudioSettingsControl.gd

```diff
@@ -50,10 +60,14 @@ func _play_sample(stream: AudioStream, player: AudioStreamPlayer2D):
 #region Node
 
 func _ready():
-	_change_bus_volume(1, "Master")
-	_change_bus_volume(0.5, "Music")
-	_change_bus_volume(0.5, "SFX")
-	_change_bus_volume(0.5, "Fauna")
+	_gm = get_node("/root/GlobalGameMaster")
+	
+	revert_changes()
+	
+	master_slider.value_changed.connect(_on_master_slider_value_changed)
```

Comment:
those sliders connect at the init of every major scene and are not disconnected at any point. I'm not sure if it can lead to an error in some weird corner case when we switch from main_menu scene to main_level or other way

Reply:
They should be different instances of both the slider and the script connecting to them, so it shouldn't be an issue.

Reply:
ok, googled a bit and they are still disconnected automatically once the scene is not in use so it should be good